src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -1,13 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { Button } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Launch } from '@edx/paragon/icons';
 import { getLaunchUrlByExamId } from '../utils';
+import messages from '../messages';
 
-const ExternalReviewDashboard = ({ exam }) => (
-  <div data-testid="review_dash">
-    {exam && <iframe title="lti_tool" src={getLaunchUrlByExamId(exam.id)} width="100%" height="1100" style={{ border: 'none' }} />}
-  </div>
-);
+const ltiToolEmbed = false; // This is set to false for now but will eventually need to be a configurable variable.
+
+const ExternalReviewDashboard = ({ exam }) => {
+  const { formatMessage } = useIntl();
+
+  // By default, launch instructor tool in a new tab. Otherwise, embed in dashboard as iframe.
+  return (
+    <div data-testid="review_dash">
+      <div style={{ padding: 10 }}>
```

Comment:
I reached out to Gabe to see if he could point me to the right place but also okay with leaving it as is as it does work! I don't know if it's overkill and better to just merge/adjust later if needed.

Reply:
in general I'd say avoid inline styling if possible but this is pretty small, let's just get this in since Proctorio devs can't launch this page w/o the fix.